guide/src/python_from_rust.md

```diff
@@ -95,22 +95,28 @@ fn main() -> PyResult<()> {
 
         // call object with PyDict
         let kwargs = [(key1, val1)].into_py_dict(py);
-        fun.call(py, (), Some(kwargs))?;
+        fun.call_bound(py, (), Some(&kwargs.as_borrowed()))?;
 
         // pass arguments as Vec
         let kwargs = vec![(key1, val1), (key2, val2)];
-        fun.call(py, (), Some(kwargs.into_py_dict(py)))?;
+        fun.call_bound(py, (), Some(&kwargs.into_py_dict(py).as_borrowed()))?;
 
         // pass arguments as HashMap
         let mut kwargs = HashMap::<&str, i32>::new();
         kwargs.insert(key1, 1);
-        fun.call(py, (), Some(kwargs.into_py_dict(py)))?;
+        fun.call_bound(py, (), Some(&kwargs.into_py_dict(py).as_borrowed()))?;
 
         Ok(())
     })
 }
 ```
 
+<div class="warning">
+
+During PyO3's [migration from "GIL Refs" to the `Bound<T>` smart pointer](./migration.md#migrating-from-the-gil-refs-api-to-boundt), `call` is temporarily named `call_bound`.
```

Comment:
Maybe this could be extend to stress that only the inherent methods on `PyAny` have these names whereas those accessible via `PyAnyMethods` start with the short names and will stay that way, i.e. I can already use the short names in a fully bound world, `call_bound` et al are necessary only for mixed code bases.

Reply:
So to be clear in this PR I don't add `call_bound` as a `PyAny` inherent method, I felt that there it's good enough for the user to do `any.as_borrowed().call(...)`. I just added `_bound` variants to `Py<T>`.

I'm open to adding `PyAny::call_bound` as a consistency piece, if you think it's desirable.

And I can extend the doc either way 👍

Reply:
Ok great, updated the doc 👍